dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java

```diff
@@ -826,10 +872,11 @@ private Workflow createOrGetWorkflow(Class workflowType, String repository, User
         Workflow workflowToUpdate = null;
         // Create workflow if one does not exist
         if (workflow.isEmpty()) {
+            // Commented this out as part of SEAB-5994
             // Ensure that a Dockstore user exists to add to the workflow
-            if (user == null) {
-                throw new CustomWebApplicationException("User does not have an account on Dockstore.", LAMBDA_FAILURE);
-            }
+            // if (user == null) {
```

Comment:
If keeping this, maybe use a feature flag (to ensure it stays compilable, maintained for code quality, etc.

Reply:
We don't have feature flags in the web service, easier to just remove. :) It'll be in the commit history if we need to restore it.